src/richie/apps/search/index_manager.py

```diff
@@ -46,7 +46,10 @@ def perform_create_index(indexable, logger=None):
     # Create the new index
     if logger:
         logger.info(f'Creating a new Elasticsearch index "{new_index:s}"...')
-    ES_INDICES_CLIENT.create(index=new_index)
+    ES_INDICES_CLIENT.create(
+        index=new_index,
+        **getattr(settings, "RICHIE_ES_CREATE_INDEX_KWARGS", {}),
+    )
```

Comment:
I would add a test for each setting to confirm that it has an effect on the underlying lib.

Reply:
I don't know how to do that.

Reply:
@sampaccoud  I've addeed a new function `new_elasticsearch_client` so it is possible to test the `ES_CLIENT` initialization. Nevertheless, I've removed the custom initialization of all the connections to the elasticsearch (all the calls to the `ES_INDICES_CLIENT` with custom kwargs). So this PR is more specific to resolve the problem.
So each installation can increase the overall timeout to the elasticsearch.